src/lsst/cm/tools/db/sqlalch_interface.py

```diff
@@ -209,6 +210,43 @@ def insert(
         self.check(new_entry.level, new_entry.db_id)
         return new_entry
 
+    def insert_step(
+        self,
+        parent_db_id: DbId,
+        config_block: str,
+        **kwargs: Any,
+    ) -> CMTableBase:
+        parent_level = parent_db_id.level()
+        campaign = self.get_entry(parent_level, parent_db_id)
+        config = campaign.config_
```

Comment:
I see where we get the fix for the input collection. Is this part the fix for setting the LSST_VERSION etc?

Reply:
it should get pulled from parent object.